### ISSUE TITLE
Containerized and reproducible phenoscape backend build

### ISSUE THREAD

**Issue by mjy** [phenoscape/KB-DataFest-2017#15]
Just throwing this out there to see where something like this might be at.

```
when I am building a new biodiversity data infrastructure
    then I want a data store that lets me persist semantic phenotypes
     and that store is "empty"
     and that store should be Docker-based (or alternative)
     and that store should be "trivially" deployable on Kubernetes
     and that store and its dependancies should be open source
     and as a bonus that store should have a minial API that lets me CRUD data
```

**Comment by hlapp** [comment]
Yes, I've been meaning to tackle this for a while, and as one possible target at the workshop, including as a basis for #10. More generally, it is currently very difficult to build the Phenoscape KB as a complete and functioning piece of infrastructure (from content ingest, to KB content build (including pre-reasoning, evolutionary branch profile construction, pre-computation of different semantic similarity metrics), to having working API and web-UI services in a reproducible manner. Having this would not only greatly lower the barrier to others instantiating their own KBs, but also greatly solidify our own infrastructure build.
